sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
```

Comment:
To change the settings, we will need to change in the deployment_settings files. We shouldnt have the option to change it directly from app code.